baseledger/ops/deploy_testnet_node.md

```diff
@@ -0,0 +1,58 @@
+https://docs.cosmos.network/master/run-node/run-node.html
+
+
+# Preparation
+
+DEVS Deploy contracts to testnet and make sure correct values configured in binaries
+DEVS run starport chain build locally for baseledger node
+DEVS run cargo build --release locally for orchestrator 
+DEVS Prepare the package of the compiled binaries to be shared with other node operators (BASELEDGER_PACKAGE)
+
+# Initialization
+
+Finspot node:
+
+1. Install latest golang
+2. Install cosmovisor: Follow the *deployment_and_chain_upgrade.md*
+3. Place binaries in the respective folders as described in *deployment_and_chain_upgrade.md*
+4. Initialize: cosmovisor init validator --chain-id=baseledger
+5. Configure genesis: Edit the genesis file for various params (voting time, tokens metadata etc.)
```

Comment:
just to add, turn off inflation, unbonding time (at least for testing), min_self_delegation should be set to 1, not sure if this is genesis

Reply:
ok, lets add during testing today